README.md

```diff
@@ -28,31 +28,134 @@ flag is highly recommended for those new to [pip](https://pip.pypa.io).
 A PEX executable (Windows not supported) and source releases are
 [here](https://github.com/planetlabs/planet-client-python/releases/latest).
 
-## Documentation
+## API Key
 
-Online documentation:
-https://planetlabs.github.io/planet-client-python/index.html
+The planet API requires an account for use. [Signup here](https://www.planet.com/explorer/?signup).
 
-Documentation is also provided for download
-[here](https://github.com/planetlabs/planet-client-python/releases/latest).
+This can be provided via the environment variable `PL_API_KEY` or the flag `-k` or `--api-key`.
 
+Using `planet init` your account credentials (login/password) can be used to obtain the api key.
 
-## Development
+## Quick Start
 
-To contribute or develop with this library, see
-[CONTRIBUTING](https://github.com/planetlabs/planet-client-python/CONTRIBUTING.md)
+### Python Library
 
+The python library is asynchronous, which greatly speeds up many interactions
+with the planet API. 
 
-## API Key
+```python
+import asyncio
+import os
+from planet.api import APlanetSession, AOrdersClient
+from planet.api.order_details import OrderDetails, Product
 
-The API requires an account for use. [Signup here](https://www.planet.com/explorer/?signup).
+API_KEY = os.getenv('PL_API_KEY')
 
-This can be provided via the environment variable `PL_API_KEY` or the flag `-k` or `--api-key`.
+image_ids = ['3949357_1454705_2020-12-01_241c']
+order_detail = OrderDetails(
+    'test_order',
+    [Product(image_ids, 'analytic', 'psorthotile')]
+)
 
-Using `planet init` your account credentials (login/password) can be used to obtain the api key.
+async def create_order(order_detail):
+    async with APlanetSession(auth=(API_KEY, '')) as ps:
+        client = AOrdersClient(ps)
+        return await client.create_order(order_detail)
 
+oid = asyncio.run(create_order())
```

Comment:
`TypeError: create_order() missing 1 required positional argument: 'order_detail'`

Reply:
Ahh shoot. I tested the code example but something must have shifted. Good catch, thanks! (and as a part of the doc switch to markdown, I am going to implement code tests in markdown docs - let me know if you have any good leads on packages for this)

Reply:
Unfortunately, I don't know of a tool that allows you to run/test Python code inside Markdown documents. :-( The solutions that I can think of right now include:
- extract the code to a Python file and run it as part of the test suite and then link to that file in the README (or copy-paste it for inline usability and put a note at the top of the Python file to include it in the README)
- use reStructuredText for the README and hopefully the `include` directive is supported (which goes along with the previous suggestion
- use the output-linking style for both, such that there would be something like: `Here is an [example script to create an order](examples/create_order.py) along with [an example of what the output might look like](examples/create_order_output)`

Unfortunately I don't know of a cleaner way to solve this that doesn't involve extracting this code to a separate Python file.

Reply:
The option of moving the long example to a separate file that can be run as a part of the test suite makes a lot of sense. We are moving to markdown for all of the documentation and there appears to be a python library (or more) that can help with doctesting python code in markdown. This work will be covered in #249.